version.txt

```diff
@@ -1 +1 @@
-13.0.0
+1.1.0
```

Comment:
Are these a remains of DSP-API's version?

Reply:
I think the one who added release-please copied from DSP-API repo but didn't adjust the version number 😉

Reply:
ok, got it